kuma/javascript/src/header/logo.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { create } from 'react-test-renderer';
-import Logo from '../../kuma/javascript/src/logo.jsx';
+import Logo from './logo.jsx';
```

Comment:
👍 to moving tests next to the code they test. As a happy side effect, it means the two files are close to each other in the pull request 😁